src/base/scene.h

```diff
@@ -0,0 +1,37 @@
+#pragma once
+
+#include <memory>
+#include <unordered_map>
+
+#include "camera.h"
+#include "mesh.h"
+#include "mesh_object.h"
+#include "scene_object.h"
+#include "tiny_gltf.h"
+
+namespace gm {
+class Scene {
+ public:
+  Scene(){};
+  Scene(const std::string &filepath);
+  std::shared_ptr<PerspectiveCamera> getCamera();
+  std::vector<std::shared_ptr<MeshObject>> meshObjects;
+  std::shared_ptr<PerspectiveCamera> camera;
+
+ private:
+  std::shared_ptr<PerspectiveCamera> loadCamera(
+      const std::vector<int> &node_ids, const tinygltf::Model &model);
+  std::shared_ptr<MeshObject> loadMeshObject(
```

Comment:
Would be nice to put some space between functions for readability

Reply:
Fixed in 1df931c